app/views/admin/invoices/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<h2>Invoice #<%= @invoice.id %></h2>
+
+<table>
+  <tr>
+    <td><%= @invoice.customer.first_name + " " + @invoice.customer.last_name  %></td>
```

Comment:
You can call customer.full_name to trim this line if you want!

Reply:
Oooo nice, I saw you had added that in the customer model. Thanks!